exporter/opentelemetry-exporter-jaeger-proto-grpc/tests/test_jaeger_exporter_protobuf.py

```diff
@@ -159,20 +159,20 @@ def test_translate_to_jaeger(self):
                 links=(link,),
                 kind=trace_api.SpanKind.CLIENT,
                 resource=Resource(
-                    attributes={"key_resource": "some_resource"}
+                    attributes={"key_resource": "some_resource"}, schema_url=""
```

Comment:
Perhaps this would look better on a new line and add the trailing comma, like you did in lines 402-403? You might also check other places where you might make the same change? 
```suggestion
                    attributes={"key_resource": "some_resource"}, 
                    schema_url="",
```

Reply:
I'm having trouble running the lint test on my machine, but pylint reports a score of 10 without the change you're suggesting and a score of 9.92 with it. I personally agree that this looks better, but I doubt that it'd pass tests. I'll look into it.

Reply:
I tried to put that line break there, but didn't have any luck getting it past the linting test.